packages/components/src/components/text-field/text-field.tsx

```diff
@@ -45,11 +45,15 @@ export class TextField {
     | 'tel'
     | 'text'
     | 'date'
+    | 'month' // example yyyy-mm
+    | 'week' // example yyyy-W##
+    | 'time' // example hh:mm
+    | 'datetime-local' // example yyyy-mm-ddThh:mm
     | 'url' = 'text';
   /** (optional) Input name */
   @Prop() name?: string = '';
   /** (optional) Input label */
-  @Prop() label: string = '';
+  @Prop() label?: string = '';
```

Comment:
I think the `label` should be mandatory because of accessibility 🤔

Reply:
hmm, that means if label is not set, then we set default as empty?

Reply:
i updated it. I remove `optional`